assets/app/scripts/services/data.js

```diff
@@ -1034,6 +1152,9 @@ angular.module('openshiftConsole')
   DataService.prototype.url = function(options) {
     if (options && options.resource) {
       var opts = angular.copy(options);
+      if(!!options.isWebsocket) {
```

Comment:
actually, I wouldn't set this... make the caller set `watch:true` in the options if they want that behavior. You'll need to adjust dataServiceSpec.js, and maybe add a test for alternate websocket uses, like following logs. something like:
```
// Watch
[{resource:'pods', namespace:"foo", isWebsocket:true, watch:true                     }, "ws://localhost:8443/api/v1/namespaces/foo/pods?watch=true"],
[{resource:'pods', namespace:"foo", isWebsocket:true, watch:true, resourceVersion:"5"}, "ws://localhost:8443/api/v1/namespaces/foo/pods?watch=true&resourceVersion=5"],

// Follow
[{resource:'pods/log', namespace:"foo", name:"mypod", follow:true, isWebsocket:true}, "ws://localhost:8443/api/v1/namespaces/foo/pods/mypod/log?follow=true"]
```